> Windows builds seem to be failing from fallout of #21248. Maybe rebase on master again and cross your fingers?

It fails for Windows only, so let's ignore it! ;-)
Ok, no jokes, it actually seems to fail during the configure stage even before the build starts:
```
21:07:17 CMake Error at C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
21:07:17   FFmpeg 5.0.0 not found, please consider using -DENABLE_INTERNAL_FFMPEG=ON
21:07:17   (missing: FFMPEG_INCLUDE_DIRS FFMPEG_LIBAVCODEC FFMPEG_LIBAVFILTER
21:07:17   FFMPEG_LIBAVFORMAT FFMPEG_LIBAVUTIL FFMPEG_LIBSWSCALE FFMPEG_LIBSWRESAMPLE
21:07:17   FFMPEG_LIBPOSTPROC) (found version "5.0.0")
21:07:17 Call Stack (most recent call first):
21:07:17   C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
21:07:17   cmake/modules/FindFFMPEG.cmake:261 (find_package_handle_standard_args)
21:07:17   cmake/scripts/common/Macros.cmake:372 (find_package)
21:07:17   cmake/scripts/common/Macros.cmake:386 (find_package_with_ver)
21:07:17   CMakeLists.txt:220 (core_require_dep)
21:07:17 
21:07:17 
21:07:17 -- Configuring incomplete, errors occurred!

```
The error is `FFmpeg 5.0.0 not found`, so it doesn't seem to be related to this PR. It seems more likely that the current Windows build is broken by the https://github.com/xbmc/xbmc/pull/21248.

--- a/addons/resource.language.en_gb/resources/strings.po
+++ b/addons/resource.language.en_gb/resources/strings.po
@@ -8875,7 +8875,7 @@ msgctxt "#15313"
 msgid "Login to RetroAchievements"
 msgstr ""
 
-#. Label for button in the in-game savestate manager
+#. Label for button which creates a new savestate in the in-game savestate manager
 #: xbmc/games/dialogs/osd/DialogInGameSaves.cpp
 msgctxt "#15314"
 msgid "Save"
@@ -8887,7 +8887,13 @@ msgctxt "#15315"
 msgid "Save progress to a new save file"
 msgstr ""
 
-#empty strings from id 15316 to 15999
+#. Label for savestates created automatically by "Autosave" function
+#: xbmc/cores/RetroPlayer/playback/ReversiblePlayback.cpp
+msgctxt "#15316"
+msgid "Autosave"
+msgstr ""
+
+#empty strings from id 15317 to 15999
 
 #: system/settings/settings.xml
 msgctxt "#16000"

--- a/addons/resource.language.en_gb/resources/strings.po
+++ b/addons/resource.language.en_gb/resources/strings.po
@@ -8878,13 +8878,13 @@ msgstr ""
 #. Label for button in the in-game savestate manager
 #: xbmc/games/dialogs/osd/DialogInGameSaves.cpp
 msgctxt "#15314"
-msgid "Save progress"
+msgid "Save"
 msgstr ""
 
-#. Caption for "Save progress" button in the in-game savestate manager
+#. Caption for "Save" button in the in-game savestate manager
 #: xbmc/games/dialogs/osd/DialogInGameSaves.cpp
 msgctxt "#15315"
-msgid "Save progress to new save file"
+msgid "Save progress to a new save file"
 msgstr ""
 
 #empty strings from id 15316 to 15999
@@ -18301,7 +18301,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#35203"
-msgid "Enable rewind if supported"
+msgid "Enable rewind (if supported)"
 msgstr ""
 
 #: system/settings/settings.xml
@@ -18407,7 +18407,7 @@ msgstr ""
 #. Title of the game OSD menu
 #: addons/skin.estuary/xml/GameOSD.xml
 msgctxt "#35221"
-msgid "Game Menu"
+msgid "Menu"
 msgstr ""
 
 #. Label of button in the in-game menu for stopping game playback
@@ -18506,10 +18506,10 @@ msgstr ""
 
 #empty strings from id 35237 to 35248
 
-#. Button to open the saved game manager from the game OSD
+#. Button to open the savestate manager from the game OSD
 #: addons/skin.estuary/xml/GameOSD.xml
 msgctxt "#35249"
-msgid "Saved games"
+msgid "Save / Load"
 msgstr ""
 
 #: xbmc/windows/GUIMediaWindow.cpp
@@ -18529,7 +18529,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#35253"
-msgid "Enable autosave if supported"
+msgid "Enable autosave (if supported)"
 msgstr ""
 
 #: system/settings/settings.xml
@@ -18573,7 +18573,7 @@ msgstr ""
 #. Button for creating a new save state from the select savestate dialog
 #: xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
 msgctxt "#35261"
-msgid "New game"
+msgid "New"
 msgstr ""
 
 #. Help text for RetroAchievements username input setting

--- a/addons/skin.estuary/xml/GameOSD.xml
+++ b/addons/skin.estuary/xml/GameOSD.xml
@@ -207,25 +207,25 @@
 								<onclick>Play</onclick>
 							</item>
 							<item id="2102">
+								<description>Save / Load button</description>
+								<label>$LOCALIZE[35249]</label>
+								<icon>osd/fullscreen/buttons/saves.png</icon>
+								<onclick>ActivateWindow(InGameSaves)</onclick>
+							</item>
+							<item id="2103">
 								<description>Reset button</description>
 								<label>$LOCALIZE[13007]</label>
 								<icon>osd/fullscreen/buttons/reset.png</icon>
 								<onclick>PlayerControl(Reset)</onclick>
 							</item>
-							<item id="2103">
+							<item id="2104">
 								<description>Stop button</description>
 								<label>$LOCALIZE[35222]</label>
 								<label2>$LOCALIZE[31060]</label2>
 								<icon>osd/fullscreen/buttons/stop.png</icon>
 								<onclick>Stop</onclick>
 							</item>
-							<item id="2107">
-								<description>Saved games button</description>
-								<label>$LOCALIZE[35249]</label>
-								<icon>osd/fullscreen/buttons/saves.png</icon>
-								<onclick>ActivateWindow(InGameSaves)</onclick>
-							</item>
-							<item id="2108">
+							<item id="2105">
 								<description>Settings button</description>
 								<label>$LOCALIZE[5]</label>
 								<icon>osd/fullscreen/buttons/settings.png</icon>

--- a/xbmc/cores/RetroPlayer/playback/ReversiblePlayback.cpp
+++ b/xbmc/cores/RetroPlayer/playback/ReversiblePlayback.cpp
@@ -19,6 +19,7 @@
 #include "games/GameServices.h"
 #include "games/GameSettings.h"
 #include "games/addons/GameClient.h"
+#include "guilib/LocalizeStrings.h"
 #include "utils/MathUtils.h"
 #include "utils/URIUtils.h"
 
@@ -134,11 +135,16 @@ std::string CReversiblePlayback::CreateSavestate(bool autosave)
 
   std::string caption = m_cheevos->GetRichPresenceEvaluation();
 
-  if (!m_autosavePath.empty())
+  if (autosave)
   {
-    std::unique_ptr<ISavestate> loadedSavestate = CSavestateDatabase::AllocateSavestate();
-    if (m_savestateDatabase->GetSavestate(m_autosavePath, *loadedSavestate))
-      label = loadedSavestate->Label();
+    if (!m_autosavePath.empty())
+    {
+      std::unique_ptr<ISavestate> loadedSavestate = CSavestateDatabase::AllocateSavestate();
+      if (m_savestateDatabase->GetSavestate(m_autosavePath, *loadedSavestate))
+        label = loadedSavestate->Label();
+    }
+    if (label.empty())
+      label = g_localizeStrings.Get(15316); // "Autosave"
   }
 
   const CDateTime nowUTC = CDateTime::GetUTCDateTime();

--- a/xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
+++ b/xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
@@ -91,7 +91,7 @@ CDialogGameSaves* CGUIDialogSelectSavestate::GetDialog(const std::string& title)
     dialog->Reset();
     dialog->SetHeading(CVariant{title});
     dialog->SetUseDetails(true);
-    dialog->EnableButton(true, 35261); // "New game"
+    dialog->EnableButton(true, 35261); // "New"
   }
 
   return dialog;

--- a/xbmc/games/dialogs/osd/DialogInGameSaves.cpp
+++ b/xbmc/games/dialogs/osd/DialogInGameSaves.cpp
@@ -31,7 +31,7 @@ CDialogInGameSaves::CDialogInGameSaves() : CDialogGameVideoSelect(WINDOW_DIALOG_
 
 std::string CDialogInGameSaves::GetHeading()
 {
-  return g_localizeStrings.Get(35249); // "Saved games"
+  return g_localizeStrings.Get(35249); // "Save / Load"
 }
 
 void CDialogInGameSaves::PreInit()
@@ -40,11 +40,11 @@ void CDialogInGameSaves::PreInit()
 
   InitSavedGames();
 
-  CFileItemPtr item = std::make_shared<CFileItem>(g_localizeStrings.Get(15314)); // "Save progress"
+  CFileItemPtr item = std::make_shared<CFileItem>(g_localizeStrings.Get(15314)); // "Save"
   item->SetArt("icon", "DefaultAddSource.png");
   item->SetPath("");
   item->SetProperty(SAVESTATE_CAPTION,
-                    g_localizeStrings.Get(15315)); // "Save progress to new save file"
+                    g_localizeStrings.Get(15315)); // "Save progress to a new save file"
 
   m_items.AddFront(item, 0);
 }

--- a/xbmc/games/dialogs/osd/DialogInGameSaves.cpp
+++ b/xbmc/games/dialogs/osd/DialogInGameSaves.cpp
@@ -53,9 +53,6 @@ void CDialogInGameSaves::InitSavedGames()
 {
   auto gameSettings = CServiceBroker::GetGameRenderManager().RegisterGameSettingsDialog();
 
-  // save current game
-  gameSettings->CreateSavestate(true);
-
   CSavestateDatabase db;
   db.GetSavestatesNav(m_items, gameSettings->GetPlayingGame(), gameSettings->GameClientID());
 

